FLT/AutomorphicForm/QuaternionAlgebra/HeckeOperators/Local.lean

```diff
@@ -0,0 +1,341 @@
+/-
+Copyright (c) 2025 Bryan Wang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Buzzard, Bryan Wang
```

Comment:
Why am I listed as an author? I'm happy for you to be the sole author -- all of this is your work.